.gitignore

```diff
@@ -24,4 +24,3 @@ dist-ssr
 *.sln
 *.sw?
 
-*storybook.log
```

Comment:
You need to add back this line. 
Basically, .gitignore needs to remain untouched.